apps/web/src/composables/coldStorage.ts

```diff
@@ -0,0 +1,28 @@
+import useLedger from '@/composables/ledger'
+import useTrezor from '@/composables/trezor'
+import { Currency, ProviderString } from '@casimir/types'
+
+export default function useWallet() {
+  const { getLedgerAddress } = useLedger()
+  const { getTrezorAddress } = useTrezor()
+
+  function getColdStorageAddress(provider: ProviderString, currency: Currency = 'ETH') {
```

Comment:
Is this used? Not sure why not async function and await getLedgerAddress and getTrezorAddress.